source/shared/cs_MicroappStructs.h

```diff
@@ -372,7 +368,7 @@ typedef union {
 	uint8_t raw[BLUENET_IPC_RAM_DATA_ITEM_SIZE];
 	// The data coming from the microapp.
 	bluenet2microapp_ipcdata_t bluenet2microappData;
-} __attribute__((packed, aligned(4))) bluenet_ipc_data_cpp_t;
+} __attribute__((packed)) bluenet_ipc_data_cpp_t;
```

Comment:
I can live with removal of the alignment for the data. However, I've one request before accepting this PR. Can we make sure it's limited in size? Something like 

```
_Static_assert(
		BLUENET_IPC_RAM_DATA_ITEM_SIZE* BLUENET_IPC_RAM_DATA_ITEMS < RAM_BLUENET_IPC_LENGTH,
		"RAM_BLUENET_IPC too small");
```

Reply:
So, when trying to use `microappRamSection`, I see that for the bootloader we have to `#include bootloader/cs_MemoryLayout.h`, while for the application it's `#include cs_MemoryLayout.h`.